<launch>
    <arg name="uav_id" default="1"/>
    <arg name="uav_name" default="uav"/>
    <group ns="uav$(arg uav_id)">
        <node pkg="sunray_uav_control" type="uav_command_pub" name="uav_command_pub" output="screen"></node>
    </group>
</launch>